tests/functional/test_newsletter.py

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+from ..pages.newsletter import NewsletterPage
+
+
+@pytest.mark.nondestructive
+def test_default_values(base_url, selenium):
+    page = NewsletterPage(base_url, selenium).open()
+    assert '' == page.email
+    assert 'United States' == page.country
+    assert 'English' == page.language
+    assert page.html_format_selected
+    assert not page.text_format_selected
+    assert not page.privacy_policy_accepted
+
+
+@pytest.mark.nondestructive
```

Comment:
I agree, what do you think @alexgibson? This would mean the test wil be skipped for sensitive URLs. I'll also add a regex to setup.cfg to indicate sensitive URLs. Would `mozilla\.(com|org)` be suitable?